#program to demonstrate Exception Handling

try:
    #a=10
    print(a)
except NameError as e:  #instead of e you can use any other variable name also
    print(e)
